src/QMCDrivers/QMCLinearOptimize.cpp

```diff
@@ -746,9 +746,8 @@ QMCLinearOptimize::RealType QMCLinearOptimize::getSplitEigenvectors(int first,
       CSF_scaled = true;
     else
       CSF_scaled = false;
-    returnValue = std::min(lowest_J_EV, lowest_CSF_EV);
+    return std::min(lowest_J_EV, lowest_CSF_EV);
   }
```

Comment:
What should the return value be if CSF_Option=="TypO!" ?

Reply:
This is an unhandled case both before and after this PR. Maybe someone else can comment what should happen - `APP_ABORT()`?

Reply:
I'll vote for that. (  Really, this points to a design problem: CSF_Option should be an enum or similar by this point so that no unknown setting is possible. Input handling would do the validation. Sadly I think there are a bunch of these throughout the code. )

Reply:
You are correct. I seem to remember several places where things are set in conditionals, with no base case fall through handling. As I said, I also shied away from going deeply into input/option handling since this has generated contentious discussion in the past, and would take quite some effort besides.